samples/crypto/sha256/README.rst

```diff
@@ -1,23 +1,25 @@
 .. _crypto_sha256:
 
-Crypto: SHA256
-##############
+Crypto: SHA-256
+###############
 
 .. contents::
    :local:
    :depth: 2
 
+The SHA-256 sample shows how to calculate and verify hashes.
```

Comment:
Maybe the term "digest" is better than "hashes"

Reply:
Closing this myself. It is only a minor NIT

Reply:
I am happy to change that to digests